go/cmd/vtctldclient/command/vreplication/lookupvindex/lookupvindex.go

```diff
@@ -368,17 +484,14 @@ func registerCommands(root *cobra.Command) {
 	// This will create the lookup vindex in the specified keyspace
 	// and setup a VReplication workflow to backfill its lookup table.
 	create.Flags().StringVar(&createOptions.Keyspace, "keyspace", "", "The keyspace to create the Lookup Vindex in. This is also where the table-owner must exist.")
-	create.MarkFlagRequired("keyspace")
```

Comment:
I think that we should unify the processing, and potentially remove the file option altogether. We can make any of these flags slices as needed. pflag has great support for specifying multiple values. IMO the current handling of flags and the file is not very intuitive.

Reply:
As discussed, we will go with the file option for now.